### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by swglad**

### CODE REVIEW

**Comment by swglad** [comment]
Figured out this one when I tried to prepare a second laptop to run sourcemash!

File: README.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ The site was built for the CS98 course at Dartmouth College.
 
 ## Installation
 1. Clone repo to computer
-2. Install all requirements from requirements.txt
+2. Install postgresql (e.g. ```brew install postgresql```)
```
Comment (alexgerstein) [inline README.md]: So postgres works now?

Comment (swglad) [inline README.md]: Not yet but sourcemash doesn't run unless postgesql is installed for now. Heroku related?

Comment (alexgerstein) [inline README.md]: It doesn't run locally if postgresql isn't installed??

**Pull request merged**